arviz/plots/ppcplot.py

```diff
@@ -38,34 +38,34 @@ def plot_ppc(
     backend=None,
     backend_kwargs=None,
     show=None,
```

Comment:
`group` must be added here somewhere.

Reply:
Ohh, what is the reason for that?

Should I add it below ```backend_kwargs```?